learning_journal/learning_journal/views.py

```diff
@@ -1,47 +1,31 @@
-from pyramid.response import Response
 from pyramid.view import view_config
-import transaction
 import datetime
-from jinja2 import Markup
-import markdown
-from sqlalchemy import update
 from .forms import EntryCreateForm, EntryUpdateForm
 from pyramid.httpexceptions import HTTPFound
 
-
-from sqlalchemy.exc import DBAPIError, ResourceClosedError, IntegrityError
-
 from .models import (
     DBSession,
     Entry,
 )
 
 
-# @view_config(route_name='list', renderer='string')
 @view_config(route_name='list', renderer='templates/pretty.jinja2')
 def list_view(request):
-    display = DBSession().query(Entry.metadata.tables['entries']).all()[::-1][:10]
+    display = DBSession.query(Entry).order_by(Entry.created)
```

Comment:
This is going to give you the item with the "smallest" date first.  That means that you'll be viewing your entries in reverse order from what you actually want.